drivers/sensor/stm32_temp/stm32_temp.c

```diff
@@ -67,13 +70,24 @@ static int stm32_temp_channel_get(const struct device *dev,
 		return -ENOTSUP;
 	}
 
+#ifdef CONFIG_SOC_SERIES_STM32L5X
+	temp = ((float)data->raw * cfg->tsv_mv) / TEMPSENSOR_CAL_VREFANALOG;
+	temp -= *TEMPSENSOR_CAL1_ADDR;
+	temp *= (TEMPSENSOR_CAL2_TEMP - TEMPSENSOR_CAL1_TEMP);
+	temp /= (*TEMPSENSOR_CAL2_ADDR - *TEMPSENSOR_CAL1_ADDR);
+	temp += 30;
```

Comment:
Magic number (30). Macro could be better replacement.

Reply:
done